codespell_lib/data/dictionary.txt

```diff
@@ -41687,7 +41715,17 @@ raedy->ready
 raelly->really
 raisedd->raised
 raison->reason, raisin,
```

Comment:
This one raises false positives with `raison d'être` or `raison d'état` which can be found in dictionaries:
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/definition/english/raison-d-etre)
* [Cambridge](https://dictionary.cambridge.org/fr/dictionnaire/anglais/raison-d-etre)
* [Collins](https://www.collinsdictionary.com/dictionary/english/raison-detat)
* [Longman](https://www.ldoceonline.com/dictionary/raison-d-etre)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/raison%20d'%C3%A9tat)

The OED even has `raison` as a variant of `reason`:
* [OED](https://www.oed.com/search/dictionary/?q=raison)

Probably enough to move it to _rare_. Since it is missing from _aspell_ dictionaries, let's leave it as is, until someone complains.